abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Case ID",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "caseId",
+          "tooltip": "Specify the case ID.",
+          "description": "Specify the case ID."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {
+        "insights": [
+          {
+            "signal": "",
+            "description": ""
+          }
+        ],
+        "eventTimeline": [
+          {
+            "event_timestamp": "",
+            "category": "",
+            "title": "",
+            "field_labels": {},
+            "ip_address": "",
+            "location": {
+              "city": "",
+              "state": "",
+              "country": ""
+            },
+            "prev_location": {
+              "city": "",
+              "state": "",
+              "country": ""
+            }
+          }
+        ]
+      }
+    },
+    {
+      "operation": "submit_false_positive_report",
+      "annotation": "submit_false_positive_report",
+      "category": "investigation",
+      "title": "Submit a Missed Attack or False Positive Report",
+      "description": "Use this to report a missed attack or a false positive judgement by Abnormal Security. We use this data to improve our models, and also give customers transparency into the frequency of Missed Attacks and False Positives.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "False Positive Report Data",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "json",
+          "name": "report_data",
+          "tooltip": "Specify the missed attack or false positive report data.",
+          "description": "Specify the missed attack or false positive report data.",
+          "value": {
+            "report_type": "false-positive",
+            "received_date": "2020/02/01",
+            "description": "Alice reported this email was missing from their inbox.",
+            "portal_link": "https://portal.abnormalsecurity.com/home/threat-center/remediation-history/1234567890"
+          }
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```

Comment:
If this action has no output and just returns a status code, Then if the status code is 200 We can return Submitted Successfully type of output

Reply:
done